exercises/practice/raindrops/.approaches/aggregate/content.md

```diff
@@ -0,0 +1,36 @@
+# `Aggregate`
+
+```csharp
+using System.Linq;
+
+public static class Raindrops
+{
+    private static readonly (int, string)[]  drips = { (3, "Pling"), (5, "Plang"), (7, "Plong") };
+
+    public static string Convert(int number)
+    {
+        var drops = drips.Aggregate("", (acc, drop) => number % drop.Item1 == 0 ? acc + drop.Item2 : acc);
+        return drops.Length > 0 ? drops : number.ToString();
```

Comment:
Maybe do:

```suggestion
        return drops.Any() ? drops : number.ToString();
```

Reply:
Well, sure, that's a possibility, as well as `drops != ""`. Since I already benchmarked as is, and since both approaches use Length > 0` I think I'll keep it for now, thanks.